openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.cpp

```diff
@@ -257,7 +258,7 @@ void TrafficMode::StartBuildingPath()
   if (m_buildingPath)
     MYTHROW(TrafficModeError, ("Path building already in progress."));
 
-  if (m_currentSegment->GetGoldenPath())
+  if (!m_currentSegment->GetGoldenPath().empty())
```

Comment:
How about to implement short method named HasGoldenPath?

Reply:
Done.